doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -4,9 +4,61 @@
 Framework Integration Guides
 #############################
 
+* :ref:`neon_intg`
 * :ref:`mxnet_intg`
 * :ref:`tensorflow_intg`
 
+.. _neon_intg:
+
+Neon |trade|
+============
+
+Use ``neon`` as a frontend
+---------------------------
+
+``neon`` is a open source Deep Learning framework specifically designed to be 
+powered by |InG| backends.
+
+.. important:: The numbered instructions below pick up from where 
+   the :doc:`install` installation instructions left off, and they presume 
+   that your system already has the library installed at ``$HOME/ngraph_dist`` as 
+   the default location. If the |nGl| code has not yet been installed to your 
+   system, you can follow the instructions on the `ngraph-neon python README`_ to 
+   install everything at once.  If the |nGl| code already is installed, 
+
+
+
+#. Set the ``NGRAPH_CPP_BUILD_PATH`` and the ``LD_LIBRARY_PATH`` path to the location 
+   where you built the nGraph libraries:
+
+   .. code-block:: bash
+
+      export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist/
+      export LD_LIBRARY_PATH=HOME/ngraph_dist/lib/       
+
+      
+#. Install the dependency for the ``neon`` framework:
+
+   .. code-block:: console
+
+      $ sudo apt-get install python3-pip 
+
+
+#. (Optionally) activate a virtualenv if you like working with virtualenvs: 
+   and go to the `python` subdirectory of the ``ngraph`` repo:
+   
+   .. code-block:: console
+
+      $ python3 -m venv .venv
+      $ . .venv/bin/activate
+      (venv)$ cd ngraph/python
+      (venv)$ pip install -U .
+
+#. See `this file`_ if you want detail about how to run unit tests. To start 
+   working with models, see the `ngraph-neon repo's README`_ to start working
+   with models. 
+
+
 .. _mxnet_intg:
 
 Compile MXNet\* with ``libngraph``
```

Comment:
Base on the latest version of ngraph_mxnet (https://github.com/NervanaSystems/ngraph-mxnet/commit/6fad67bf64bab7bc115e76fcd185ff877e2045a2) , we need to fix the  Compile MXNet* with libngraph:


a. Part 4:
USE_NGRAPH option (line -80-) to true with 1 and set the NGRAPH_DIR (line -81-).
Line numbers change to : 
USE_NGRAPH option (line 100) to true with 1 and set the NGRAPH_DIR (line 101)

b. Part 5:
USE_MKL2017 (line -93-) and USE_NNPACK (line -100-).
Line numbers change to : 
USE_MKL2017 (line 113) and USE_NNPACK (line 120).

Reply:
fixed